pkg/collector/py/loader.go

```diff
@@ -49,9 +48,21 @@ func init() {
 	loaders.RegisterLoader(10, factory)
 
 	configureErrors = map[string][]string{}
-	py3Warnings = map[string][]string{}
+	py3Warnings = map[string]string{}
 	pyLoaderStats = expvar.NewMap("pyLoader")
 	pyLoaderStats.Set("ConfigureErrors", expvar.Func(expvarConfigureErrors))
+	pyLoaderStats.Set("Py3Warnings", expvar.Func(expvarPy3Warnings))
+
+	agentVersionTags = []string{}
+	if agentVersion, err := version.New(version.AgentVersion, version.Commit); err == nil {
+		agentVersionTags = []string{
+			fmt.Sprintf("agent_version_major:%d", agentVersion.Major),
+			fmt.Sprintf("agent_version_minor:%d", agentVersion.Minor),
+			fmt.Sprintf("agent_version_bugfix:%d", agentVersion.Patch),
```

Comment:
we should likely call this tag `agent_version_patch`, to stick to the `semver` naming